client/src/app/assignments/page.tsx

```diff
@@ -26,7 +62,18 @@ export default function Assignments() {
 
   return (
     <div style={{ display: "flex", flexDirection: "column" }}>
-      <div>
+      {assignments.map((assignment: Assignment) => {
+        return (
+          <div
+            key={assignment.musician.musician_id}
+            className={classes.container}
+          >
+            <ItemCard item={assignment.musician} />
+            <ItemCard item={assignment.instrument} />
+          </div>
+        );
+      })}
+      <div className={classNames(classes.buttonContainer)}>
```

Comment:
I think we actually don't need to wrap this with classNames? I believe we really only need to use it if we are trying to use more than one class, or if dealign with conditional styling.

Reply:
Got it! I'll go through them and get rid of the wrapper.